### ISSUE TITLE
[import/no-duplicates] (prefer-inline) Does not detect across type and regular imports

### ISSUE THREAD

**Issue by benkrejci** [import-js/eslint-plugin-import#2834]
I misread the [no-duplicates](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-duplicates.md) doc to mean that when using the "prefer-inline" option, the following would be detected as an error:
```ts
import type { Type } from "mod";
import { value } from "mod";
```
And would fix to:
```ts
import { type Type, value } from "mod";
```

I realize now that this is not what is stated in the doc, but I still want this behavior. It is possible to implement, and from my perspective is probably what those using `prefer-inline` actually want. But if not, perhaps could be implemented with a new option, e.g. `prefer-combined-type-imports`

**Comment by benkrejci** [comment]
https://github.com/import-js/eslint-plugin-import/pull/2835

**Comment by kevinbosman** [comment]
This took me by surprise too. The docs say:
> In order to support fixing to an inline type import when duplicate imports are detected, `prefer-inline` can be set to true.

I find it hard to read this in any way other than how @benkrejci also interpreted it. Earlier in the same document a duplicate is defined as "have to point to the same module on the filesystem". Surely a 2-line import as posted above should classify as a "duplicate import" using this definition?

If I'm also reading this incorrectly, an edit to the documentation to clarify would be greatly appreciated. Thanks!

**Comment by ljharb** [comment]
I definitely agree that when `prefer-inline` is set, only one `from 'mod'` should remain.

**Comment by benkrejci** [comment]
Since there seems to be some agreement here, I added tests and an example to the doc: https://github.com/import-js/eslint-plugin-import/pull/2835

**Comment by benkrejci** [comment]
@ljharb hey any idea when the next release will be? (Or maybe we could get an `-alpha` release 🙏). Happy to help if I can, thanks.

**Comment by ljharb** [comment]
I don't do prereleases, and I don't have planned release dates, but I am hoping to get one done soon.
